src/pages/events/[id]/repair-requests.tsx

```diff
@@ -109,8 +112,8 @@ export default function RepairRequests() {
                 <Modal showModal={eventModal} setShowPopup={showEventModal}>
                   {" "}
                   <div className="text-center">
-                    <h1 className="text-xl font-bold">New Event Form</h1>
-                    <div>
+                    <h1 className="text-xl font-bold">Add a repair request</h1>
+                    <div className="max-w-full">
                       <RepairAttemptForm></RepairAttemptForm>
```

Comment:
Since we're on the event page, when a repair request is created on this page, it should be added to the event. You'll most likely need to pass down the event id as a prop